content/en-US/api/CSSAnimation/CSSAnimation.md

```diff
@@ -0,0 +1,55 @@
+---
+recipe: api-feature
+title: 'CSSAnimation'
+mdn_url: /en-US/docs/Web/API/CSSAnimation
+specifications: https://drafts.csswg.org/css-animations-2/#the-CSSAnimation-interface
+browser_compatibility: api.CSSAnimation
+
+---
+
+## Description
+
+The `CSSAnimation` interface of the [CSS Animations Level
+2](https://drafts.csswg.org/css-animations-2/) represents a CSS animation as an
+[`Animation`](/en-US/docs/Web/API/Animation/Animation) object in the
+[Web Animations API](/en-US/docs/Web/API/Web_Animations_API) model.
+
+As such it extends the [`Animation`](/en-US/docs/Web/API/Animation/Animation)
+interface and provide playback controls and the ability to change various
+aspects of the CSS animation.
```

Comment:
```suggestion
The `CSSAnimation` interface of the [Web Animations API](/en-US/docs/Web/API/Web_Animations_API) provides playback controls and the ability to change various
aspects of the CSS animation.
```

Reply:
Inheritance and spec links are handled in a different (and standard) way on MDN. The spec section is not in the boilerplate because I build it from the metadata in a script. You can [see the result here](https://developer.mozilla.org/en-US/docs/Web/API/AbsoluteOrientationSensor#Specifications).

Reply:
Just a note that CSSAnimation interface is defined in CSS Animations Level-2 and extends the Animation interface that is defined Web Animations API. So I am not sure exactly "interface the [spec]" means in this context.

Just wanted to mention this but will leave its interpretation up to you. :)

Reply:
Your intro describes what CSSAnimation inherits from and provides a link to the spec. I simply meant that both bits of information are handled by different but standard ways.